releases.go

```diff
@@ -315,6 +321,17 @@ func newSchedulerProcess(release *Release, name string, p Process) *scheduler.Pr
 		"empire.app.process": name,
 	}
 
+	var exposure *scheduler.Exposure
+	// For `web` processes defined in the standard procfile, we'll
+	// generate a default exposure setting and also set the PORT
+	// environment variable for backwards compatability.
+	if name == webProcessType && len(p.Ports) == 0 {
+		exposure = standardWebExposure(release.App)
+		env["PORT"] = "8080"
+	} else {
+		exposure = processExposure(release.App, p)
```

Comment:
Is there no way to share the port info?  SEems like we should keep the env var around, no matter what, if at all possible.  Users may be relying on it.